manifest_test.go

```diff
@@ -158,58 +144,6 @@ func buildTable(t *testing.T, keyValues [][]string, bopts table.Options) *table.
 	return tbl
 }
 
-func TestOverlappingKeyRangeError(t *testing.T) {
-	dir, err := ioutil.TempDir("", "badger-test")
```

Comment:
Perhaps instead of deleting we can skip this test?

Reply:
done